dev.neovide.neovide.yml

```diff
@@ -0,0 +1,233 @@
+id: dev.neovide.neovide
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: neovide
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
why is this needed? This is not allowed by default.

Reply:
Oh, hmm the original exception from the previous PR is still there https://github.com/flathub-infra/flatpak-builder-lint/pull/336/

Reply:
`--talk-name=org.freedesktop.Flatpak` needed to run host neovim instance (see readme.md), and to acess host shell from neovim terminal emulator (`:term host-spawn`)

org.vim.Vim and com.vscodium.codium packages also uses this permission for terminal emulating


and about fallback-x11, ipc, dri permissions, i don't really know why this permissions needed for, this manifest was originally written by @sqwxl, so probably we should ask them, i didn't touch any permissions